### PR TITLE
Background color change of canvas solved

### PR DESCRIPTION
**Pull request by diyashah28**
# Issue Title: Background color is not applied to the canvas on selecting the color

https://github.com/singodiyashubham87/Draw-it-out/assets/145260436/19607bee-3ece-4d39-b6b4-e450ce95434c

*resolve issue:* #224 

## Type of change ☑️

What sort of change have you made:
I have fixed the bug of background color change of canvas. The background color of canvas did not change on selecting a color from color picker on the left side panel.

 
## How Has This Been Tested? ⚙️

I have attached a video showing the color change of canvas on clicking the color picker box on the side panel

### CODE REVIEW

**Comment by singodiyashubham87** [comment]
@diyashah28 Merge the main branch & push commit.

**Comment by diyashah28** [comment]
It is saying that merging is blocked and can be performed after an approving review.

**Comment by diyashah28** [comment]
Also there is no label assigned to this pull request.

**Comment by singodiyashubham87** [comment]
@diyashah28 Ignore that warning, Just sync your local branch with the main branch.

**Comment by diyashah28** [comment]
@singodiyashubham87 I have merged the main branch.

File: package.json
Comment (singodiyashubham87) [inline package.json:1]: What was the need of this change?

Comment (diyashah28) [reply package.json:1]: @singodiyashubham87  I used the command npm init -y which updated the package.json file and added any missing fields.

**Comment by singodiyashubham87** [comment]
@diyashah28 Any updates?

**Comment by diyashah28** [comment]
@singodiyashubham87 I have already made the changes regarding the issue and merged with the main branch. Those changes are final.

**Comment by singodiyashubham87** [comment]
@diyashah28 
image

**Comment by singodiyashubham87** [comment]
@diyashah28 Any updates?

**Comment by singodiyashubham87** [comment]
Closing due to inactivity!

**Pull request closed without merging**